src/tests/integration_test/constants.cljs

```diff
@@ -5,3 +5,10 @@
 (def community {:membership 1 :name "foo" :description "bar"})
 
 (def account-name "account-abc")
+
+(def recovery-account
+  {:name        "wallet-abc"
+   :seed-phrase "destroy end torch puzzle develop note wise island disease chaos kind bus"
+   :key-uid     "0x6827f3d1e21ae723a24e0dcbac1853b5ed4d651c821a2326492ce8f2367ec905"
+   :public-key  "0x48b8b9e2a8f71e1beae729d83bcd385ffc6af0d5"
+  })
```

Comment:
public key is not needed but handy to have easily available so we can ensure the balance is alright etc